website/content/en/docs/faq.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "FAQs"
+linkTitle: "FAQs"
+weight: 30
+---
+
+## General
+### How does a Provisioner decide to manage a particular node?
+Each node will have a set of predetermined Karpenter labels. Provisioners will use `name` and `namespace` labels to distinguish between Provisioners. Furthermore, a Provisioner will only take action on a node based on the label that details what phase a node is in. e.g. a provisioner will only consider a node for termination if its phase label says "underutilized".
```

Comment:
Can we be more specific about the labels that Karpenter will add to nodes?

Reply:
i'll prep an example tomorrow, unless @njtran has an example before then? :)

Reply:
The only labels that Karpenter will add into a node are listed [here](https://github.com/awslabs/karpenter/blob/main/pkg/apis/provisioning/v1alpha3/provisioner.go#L120-L126). The do-not-evict annotation will not be added by Karpenter but is respected in eviction.

Reply:
Additionally, we will add the finalizer, listed in the same file [here](https://github.com/awslabs/karpenter/blob/main/pkg/apis/provisioning/v1alpha3/provisioner.go#L133).

Also, any constraints specified in the provisioner will be added to the nodes on allocation.

Reply:
Can't suggest here, I'm going to resolve and then add what i had in mind in the current file